org_chart.js

```diff
@@ -1,73 +1,58 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
+//CONSTRUCTOR AND PROTOTYPE////////////////
+function Hire(name, position, project, directory, username = name.replace(' ', '.').replace('-', '_').toLowerCase()) {
+    this.name = name;
+    this.email = username + '@musky.com';
+    this.position = position;
+    this.project = project;
+    this.skills = [];
+
+    directory.push(this);
+    if (this.position != 'Boss') {
+        console.log(`${this.name} is a ${this.position} working on ${this.project}`);
+        console.log(`and can be reached at ${this.email}`);
```

Comment:
Hmmm....  This is assuming that you want the introduction to happen only (and always) when the person is created.  This is a code smell to me.  Your constructor is more and more doing stuff that are strictly not construction of the object.

Having special conditionals for excluding a subset (in this case, the Boss) is another major code smell.  It suggests that your OOP design is flawed.  When we cover Polymorphism, I can suggest a better solution.  Notice how in my solution, you don't have to check for the Boss.